docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '1'
```

Comment:
I get this error locally:
<img width="1196" alt="Screen Shot 2023-06-30 at 9 51 28 AM" src="https://github.com/NYPL/research-catalog/assets/1280564/65d91743-6b03-4fc0-9b70-b681e8a0c5e9">

Try version `'2'`, that works for me. Looks like v1 won't be supported anymore https://docs.docker.com/compose/compose-file/compose-versioning/

Reply:
Or version 3. I only just tried 2 but maybe 3 is the way to go.

Reply:
Good call! Updated to version 3.